docs/reference/setup/install/docker.asciidoc

```diff
@@ -360,6 +360,24 @@ IMPORTANT: The container **runs {es} as user `elasticsearch` using
 uid:gid `1000:0`**. Bind mounted host directories and files must be accessible by this user,
 and the data and log directories must be writable by this user.
 
+[[docker-keystore-bind-mount]]
+===== Mounting an {es} keystore
+
+By default, {es} will auto-generate a keystore file for secure settings. This file
+is obfuscated but not encrypted. If you want to encrypt your <<secure-settings,secure settings>>
+by password-protecting the keystore that stores them, you must use the `elasticsearch-keystore`
```

Comment:
```suggestion
with a password, you must use the `elasticsearch-keystore`
```

nit: it just felt we said "password-protect" too many times in a row :)

Reply:
Truth.